src/NServiceBus.Core/Hosting/StartupDiagnostics/StartupDiagnosticEntries.cs

```diff
@@ -6,16 +6,14 @@ class StartupDiagnosticEntries
     {
         public void Add(string sectionName, object section)
         {
-            entries.Add(new StartupDiagnosticEntry
+            Entries.Add(new StartupDiagnosticEntry
             {
                 Name = sectionName,
                 Data = section
             });
         }
 
-        public IEnumerable<StartupDiagnosticEntry> Entries => entries;
-
-        List<StartupDiagnosticEntry> entries = new List<StartupDiagnosticEntry>();
+        internal List<StartupDiagnosticEntry> Entries = new List<StartupDiagnosticEntry>();
```

Comment:
should this be a property?

Reply:
Properties provide encapsulation when exposing on a public api.  Ie we can change the get/set without the consumer recompiling. Makes no sense for an internal given we can change to a prop and change get/set without breaking any contract.

Reply:
still feels kinda awkward to have the field in pascal casing if it's not public. I'd rather see it be either a property or changed to lowercase. but no blocker.

Reply:
indifferent. happy for @Particular/nservicebus-maintainers to revert this change or go with a prop.

Reply:
I'm indifferent as well